packages/expo-constants/src/Constants.types.ts

```diff
@@ -199,6 +208,9 @@ export interface NativeConstants {
    * @hidden
    */
   supportedExpoSdks?: string[];
+  /**
+   * Returns the platform-specific manifest object.
+   */
```

Comment:
```suggestion
  /**
   * Returns the platform-specific platform manifest object. 
   */
```

(somehow we should probably communicate that this is an independent concept from `manifest`/`manifest2` manifests, no idea how to do that cleanly)

Reply:
@wschurman, based on your feedback, I've updated it to the following and also added a callout to note the point you mentioned:

```ts
  /**
   * Returns the specific platform manifest object.
   *
   * > **Note**: This is distinct from the `manifest` and `manifest2`.
   */
```

Let me know if you've any further feedback.